core/src/main/scala/neotypes/mappers.scala

```diff
@@ -13,7 +13,7 @@ import scala.util.Try
 object mappers {
   @annotation.implicitNotFound("Could not find the ResultMapper for ${A}")
   trait ResultMapper[A] { self =>
-    def to(value: List[(String, Value)], typeHint: Option[TypeHint]): Either[Throwable, A]
+    def to(value: List[(String, Value)], typeHint: Option[TypeHint], errors: Seq[Throwable] = Nil): Either[Throwable, A]
```

Comment:
I believe that default value is not needed at all.
Apply to all the following ones.